project-templates/node-json-vscode/README.md

```diff
@@ -0,0 +1,102 @@
+# Eclipse GLSP - Project Template: 🖥️ Node ● 🗂️ Custom JSON ● 🖼️ VS Code
+
+This folder contains a simple *project template* to get you started quickly for your diagram editor implementation based on [GLSP](https://github.com/eclipse-glsp/glsp).
+It provides the initial setup of the package architecture and environment for a GLSP diagram editor that uses ...
+
+*   🖥️ The [Node-based GLSP server framework](https://github.com/eclipse-glsp/glsp-server-node)
+*   🗂️ A custom JSON format as source model
+*   🖼️ The [VS Code integration](https://github.com/eclipse-glsp/glsp-vscode-integration) to make your editor available in VS Code
+
+To explore alternative project templates or learn more about developing GLSP-based diagram editors, please refer to the [Getting Started](https://www.eclipse.org/glsp/documentation/gettingstarted) guide.
+
+## Project structure
+
+This project is structured as follows:
+
+*   [`glsp-client`](glsp-client)
+    *   [`tasklist-glsp`](glsp-client/tasklist-glsp): diagram client configuring the views for rendering and the user interface modules
+    *   [`tasklist-vscode`](glsp-client/tasklist-vscode): glue code for integrating the editor into VS Code
+        *   [`extension`](glsp-client/tasklist-vscode/extension): VS Code extension responsible for starting the glsp-server and registering the `webview` as a custom editor
+        *   [`webview`](glsp-client/tasklist-vscode/webview): integration of the `tasklist-glsp` diagram as webview
+    *   [`workspace`](glsp-client/workspace): contains an example file that can be opened with this diagram editor
+*   [`glsp-server`](glsp-server)
```

Comment:
Maybe we should also add a short description to the `glsp-server` item`

Reply:
Yeah, I wasn't really sure what to write with only a few words that isn't obvious, that's why I skipped it. If you have an idea, feel free to add something.